bot/exts/help_channels/_stats.py

```diff
@@ -22,21 +22,21 @@ def report_counts() -> None:
             log.warning(f"Couldn't find category {name!r} to track channel count stats.")
 
 
-async def report_complete_session(channel_id: int, is_auto: bool) -> None:
+async def report_complete_session(channel_id: int, closed_on: str) -> None:
     """
     Report stats for a completed help session channel `channel_id`.
 
     Set `is_auto` to True if the channel was automatically closed or False if manually closed.
     """
-    caller = "auto" if is_auto else "command"
+    caller = f"auto.{closed_on}" if closed_on else "command"
```

Comment:
this makes sure that auto is not prepended if the channel is closed by commend. 

alternatively, get rid of this prepending entirely and explicitly pass "auto.cleanup" etc.

```suggestion
    caller = f"auto.{closed_on}" if closed_on != "command" else "command"
```